### DISCUSSION TITLE
Hacked

### DISCUSSION THREAD

**Discussion by Memerdz** [SpotX-Official/SpotX#340] [General]
I received an email that my password was changed just 5 minutes after the installation, I thought it was safe because I used it before, I immediately recovered the account and uninstalled

**Comment by jetfir3** [comment]
There's nothing in the code remotely related to grabbing spotify authentication. Nothing setup to send data to an external server. Nothing compiled or obfuscated to hide anything. 

If this password change event is true, it's entirely coincidence that it happened after applying SpotX. The script/code is wide open for all to read and review. 

This statement assumes that SpotX code was run directly from this repo or the official githack mirror. Any code/script/etc that isn't sourced directly from either official git repo or official telegram channel/group may not be safe.

**Comment by Iheuzio** [comment]
If you logged in on a session for a browser tab, chances are you got your token stolen depending on what was open in it. Malwarebytes flags this yes, but its a false positive

**Comment by xanderusa** [comment]
Not only Malwarebytes but also Virustotal. It's flagged.

**Comment by Lord-Noodle** [comment]
Guys this will Steal your password so, **DONT INSTALL!**

It happened to aswell as mine got changed after few min so, not coincidence since i have antivirus, vpn and password manager to create random strong passwords which means it steals/exposes your session token.

I guess 2 things is happening:

1. They really want to steal your account for idk what reason.
2. There might be backdoor that they don't know.

**Reply by amd64fox** [reply]
why am I not surprised

![2024-09-14 083823](https://github.com/user-attachments/assets/85210d41-de4d-462b-bad3-43883e579722)

**Reply by Lord-Noodle** [reply]
<img width="1228" alt="Screenshot 2024-09-14 113919" src="https://github.com/user-attachments/assets/a21bd0ca-2f88-4b3e-bce7-0e1b1aa2ba9e">

This is what i got after installing it after couple of minutes

**Reply by Lord-Noodle** [reply]
Look im not here to defame your product but this is what happened to me. I made new github acc cuz my old one was hacked but i didn't bother recovering the old one.

Im warning others due to my experience so, not sure what your intentions are but if you it's good intentions, might check if malicious people have access.

**Reply by amd64fox** [reply]
wow, your password was reset, that's something! Although at first, you said it had been changed, turns out you lied, strange...

Before giving advice to others, work on yourself first. You're clearly not technically proficient, yet for some reason, you're giving advice without fully understanding your own situation. 

Focus on self-education, read relevant materials, take courses, and surround yourself with people who are more knowledgeable than you

btw, learn how to use a vpn, because your password will keep getting reset, whether you use SpotX or not.

cheers

**Reply by Lord-Noodle** [reply]
Fine, then please tell me why it was flagged as suspicious activity after installing spotx?

I have no reason to lie because people tend to panic when things like this happen.

**Reply by amd64fox** [reply]
I already told you the reason above, it happens in 99% of cases because of this, you ask questions in the wrong place, do you want to use spotify without your panic attacks? here is the [solution](https://support.spotify.com/us/article/payment-methods/), if you have more questions then I suggest you go to their [forum](https://community.spotify.com/) and get official support.

**Comment by pWoLiAn** [comment]
hmm what was so special abt ur account that they stole urs but not mine
